Common/include/CConfig.hpp

```diff
@@ -6396,6 +6396,13 @@ class CConfig {
    */
   bool GetSolid_Wall(unsigned short iMarker) const;
 
+  /*!
+   * \brief Determines whether a marker with index iMarker is an inlet, outlet or far-field.
+   * \param iMarker
+   * \return <TRUE> it marker with index iMarker is a solid boundary.
+   */
+  bool Getinoutfar(unsigned short iMarker) const;
+
```

Comment:
We use large arrays to determine if iPoint is on a wall. I copied this procedure. This results in a boolean array of size NPoints. Maybe better to use a smarter data structure for this.